TEST_APPS/testcases/mesh_minimal_startup.py

```diff
@@ -63,7 +63,7 @@ def case(self):
         # 2. Verify 6LoWPAN bootstrap (fails to timeout or bootstrap failure trace)
         bootstrap_complete = "Connected. IP ="
         bootstrap_fail = "Connection failed!"
-        loops = 105 # Connection can take up a long time (>1020 seconds in case of Wi-SUN).
+        loops = 115 # Connection can take up a long time (>1020 seconds in case of Wi-SUN).
```

Comment:
@mikter is this long enough timeout that you are interested to investigate if this fails?